### DISCUSSION TITLE
db error: FATAL: SASL authentication failed

### DISCUSSION THREAD

**Discussion by aanomm** [umami-software/umami#1302] [Q&A]
Hi All

- I'm new to Unami, Superbase, and Vercel, so I'm not surprised my first Umami install (using that combo) hasn't worked 
- Does the error below narrow down what I might be doing wrong?

```
[POST] /api/auth/login
23:42:41:96
2022-07-15T13:42:43.409Z	496b5cd7-f253-475e-9e44-73628aadb0c3	ERROR	PrismaClientInitializationError: Error querying the database: db error: FATAL: SASL authentication failed
    at RequestHandler.request (/var/task/node_modules/@prisma/client/runtime/index.js:49028:15)
    at async PrismaClient._request (/var/task/node_modules/@prisma/client/runtime/index.js:49919:18)
    at async __WEBPACK_DEFAULT_EXPORT__ (/var/task/.next/server/pages/api/auth/login.js:85:21)
    at async Object.apiResolver (/var/task/node_modules/next/dist/server/api-utils/node.js:182:9)
    at async NextNodeServer.runApi (/var/task/node_modules/next/dist/server/next-server.js:386:9)
    at async Object.fn (/var/task/node_modules/next/dist/server/base-server.js:488:37)
    at async Router.execute (/var/task/node_modules/next/dist/server/router.js:228:32)
    at async NextNodeServer.run (/var/task/node_modules/next/dist/server/base-server.js:600:29)
    at async NextNodeServer.handleRequest (/var/task/node_modules/next/dist/server/base-server.js:307:20)
    at async Server.module.exports (/var/task/___next_launcher.cjs:29:9) {
  clientVersion: '3.15.2',
  errorCode: undefined
}
```

**Comment by hsb-tonmoy** [comment]
Were you able to find a solution to this?

**Comment by JMPerez** [comment]
I'm running into this same issue using Prisma + Supabase. Any solution?

**Reply by hsb-tonmoy** [reply]
> I'm running into this same issue using Prisma + Supabase. Any solution?

In my case, the problem was a non-URL friendly Postgress password. I ended up choosing a 32 character alphanumeric password and made sure that the URL is correct.

**Reply by webdiego** [reply]
> > I'm running into this same issue using Prisma + Supabase. Any solution?
> 
> In my case, the problem was a non-URL friendly Postgress password. I ended up choosing a 32 character alphanumeric password and made sure that the URL is correct.

Yes It works, I had a database psw like 23je^3k , I changed the psw by removing the ^

**Reply by gualberto2** [reply]
Hi guys I am very new, I am having the same issue above using Clerk for user Auth, Prisma as an ORM, and supabase for the database. Could you elaborate on the 32 character alphanumeric password and provide an example? I would like to try any new solutions to my problem.

**Reply by gualberto2** [reply]
I feel dumb haha, I just entered the 32 character password into my password I was confuse by "23je^3k"  and realized PSW stands for password (I hope lol) but doing so fixed the error....

TLDR: make password 32 characters long (alphanumerical)
